### DISCUSSION TITLE
Feature request - Ability to get the performance graph for networth

### DISCUSSION THREAD

**Discussion by sujan46** [ghostfolio/ghostfolio#3503] [Ideas]
Hello Team,

Is it possible to include a performance graph for networth like we have for investments?

Thank you

**Comment by dtslvr** [comment]
Hello @sujan46

Your idea seems to be a duplicate of https://github.com/ghostfolio/ghostfolio/discussions/2641. Please join the discussion there and use the upvote functionality, thank you.
